hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -127,18 +127,31 @@ public JobResult getJobResult(long jobId) {
     }
 
     void checkQuorumValues() {
-        int currentQuorumSize = getQuorumSize();
-        for (JobRecord jobRecord : jobRepository.getJobRecords()) {
-            if (jobRecord.getConfig().isSplitBrainProtectionEnabled()) {
-                if (currentQuorumSize > jobRecord.getQuorumSize()) {
-                    logger.warning("Current quorum size: " + currentQuorumSize
-                            + " of cluster is larger than quorum size: " + jobRecord.getQuorumSize() + " of job "
-                            + idToString(jobRecord.getJobId()));
+        if (!shouldCheckQuorumValues()) {
+            return;
+        }
+
+        try {
+            int currentQuorumSize = getQuorumSize();
+            for (JobRecord jobRecord : jobRepository.getJobRecords()) {
+                if (jobRecord.getConfig().isSplitBrainProtectionEnabled()) {
+                    if (currentQuorumSize > jobRecord.getQuorumSize()) {
+                        logger.warning("Current quorum size: " + currentQuorumSize
+                                + " of cluster is larger than quorum size: " + jobRecord.getQuorumSize() + " of job "
+                                + idToString(jobRecord.getJobId()));
+                    }
                 }
             }
+        } catch (Exception e) {
+            logger.fine("check quorum values task failed", e);
         }
     }
 
+    private boolean shouldCheckQuorumValues() {
```

Comment:
I don't like these single-use private methods just for the sake of using method name as a comment, but it seems everyone else does...

Reply:
I understand your concern, but writing single-use methods for checking some conditions is a good use case.